build.gradle

```diff
@@ -21,15 +20,25 @@ allprojects {
     mavenCentral()
     jcenter()
   }
+
+  group = 'io.arrow-kt'
+
+  //When updating version it must be also changed in ArrowKotlinGradleSubplugin#VERSION
+  version = '0.0.1'
 }
 
-task buildMeta {
-  dependsOn 'clean'
-  dependsOn 'build'
+task deployMeta {
+  dependsOn ':compiler-plugin:publishToMavenLocal'
   dependsOn ':gradle-plugin:publishArrowPluginMarkerMavenPublicationToMavenLocal'
   dependsOn ':gradle-plugin:publishPluginMavenPublicationToMavenLocal'
-  tasks.findByPath(':gradle-plugin:publishArrowPluginMarkerMavenPublicationToMavenLocal').mustRunAfter 'build'
-  tasks.findByPath(':gradle-plugin:publishPluginMavenPublicationToMavenLocal').mustRunAfter 'build'
+  dependsOn ':gradle-plugin:publishToMavenLocal'
+}
+
+task buildMeta {
+  dependsOn 'build'
```

Comment:
Can we add `clean`, too?

Reply:
I don't like `clean`, because it makes build much longer. I understand using it in specific times like changing branch, or updating gradle - but not during every build.

Reply:
CI needs it. I don't mind having it on your settings, but that introduces changes to our process.